src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.builds

```diff
@@ -3,10 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Microsoft.Win32.Primitives.csproj">
-      <OSGroup>Linux</OSGroup>
-    </Project>
-    <Project Include="Microsoft.Win32.Primitives.csproj">
-      <OSGroup>OSX</OSGroup>
+      <OSGroup>Unix</OSGroup>
```

Comment:
I made changes until the nuspec diff looked correct.  I'll go back and clean it up some more.